### DISCUSSION TITLE
Dia

### DISCUSSION THREAD

**Discussion by JuliaAgostino** [mayrinck/FOSSonMicrosoftStore#15] [Requests] [answered]
Dia website: http://dia-installer.de/

Dia privacy policy: http://dia-installer.de/privacypolicy.html

Logo: https://upload.wikimedia.org/wikipedia/commons/f/f1/Dia.svg

Description: "Dia (/ˈdiːə/) is free and open source general-purpose diagramming software, developed originally by Alexander Larsson. Dia uses a controlled single document interface (SDI) similar to GIMP and Inkscape."

**Comment by mayrinck** [comment, marked as answer]
Dia is now in the store, didn't give much updates about this one around here but here you go!
https://www.microsoft.com/en-us/p/dia-diagram-editor/9npvgp9l96jj

**Reply by JuliaAgostino** [reply]
> Dia is now in the store, didn't give much updates about this one around here but here you go!
> https://www.microsoft.com/en-us/p/dia-diagram-editor/9npvgp9l96jj

That's great to hear! Thank you so much!
